.circleci/config.yml

```
@@ -51,49 +51,57 @@ commands:
       - run:
           name: Upload coverage to Codecov
           command: pipenv run codecov
+  lint:
+    description: "Lint python files"
+    steps:
+      - checkout
+      - run:
+          name: Install dependencies
+          command: pipenv install
+      - run:
+          name: Install pylint
+          command: |
+            pipenv run pip install pylint
+      - run:
+          name: Run pylint
+          command: |
+            pipenv run pylint -sn --disable=R gvmtools
```

```suggestion
            pipenv run pylint --score=n --disable=R gvmtools
```
I would suggest using long parameters to make the meaning more obvious.